### PR TITLE
Hardware: add rocketchip-zcu102 instructions

### PR DESCRIPTION
**Pull request by chrisguikema**

### CODE REVIEW

File: Hardware/rocketchip-zcu102.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+cmake_plat: rocketchip-zcu102
+xcompiler_arg: -DRISCV64=1
+platform: Rocketchip
+arch: RV64IMAFDC
+virtualization: "Yes"
+iommu: "No"
+simulation_target: false
+Status: Unverified
+Contrib: DornerWorks
+Maintained: seL4 Foundation
```
Comment (axel-h) [inline Hardware/rocketchip-zcu102.md:12]: I think for now you guys have to act as maintenance contact also.
```suggestion
Maintained: DornerWorks
```

Comment (chrisguikema) [reply Hardware/rocketchip-zcu102.md]: Done.

**Review by axel-h — APPROVED** [review]

**Comment by axel-h** [comment]
Looks ok to me. But I can't test the instructions.

File: Hardware/rocketchip-zcu102.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
+---
+riscv_hardware: true
+cmake_plat: rocketchip-zcu102
+xcompiler_arg: -DRISCV64=1
+platform: Rocketchip
+arch: RV64IMAFDC
+virtualization: "Yes"
```
Comment (axel-h) [inline Hardware/rocketchip-zcu102.md:7]: I wonder, can't we just give this as boolean also - or does the parser fail then?
```suggestion
virtualization: true
iommu: false
```

**Review by kent-mcleod — APPROVED** [review]

**Pull request merged**